content/docs/dvclive/index.md

```diff
@@ -8,15 +8,7 @@ fully compatible with DVC. You can
 <cards>
 
   <card href="/doc/dvclive/get-started" heading="Get Started">
-    A step-by-step introduction into basic DVCLive features
-  </card>
-
-  <card href="/doc/dvclive/dvclive-with-dvc" heading="DVCLive with DVC">
-    Discover how DVCLive and DVC can integrate in several useful ways
-  </card>
-
-  <card href="/doc/dvclive/ml-frameworks" heading="ML Frameworks">
-    Use DVCLive alongside your favorite ML Framework
+    A step-by-step introduction
   </card>
```

Comment:
Not sure I get why we removed these 2 from the docs home page if that's what we want to focus on.